externs/v20140218_eme.js

```diff
@@ -0,0 +1,108 @@
+/**
```

Comment:
To keep with the existing convention (externs/mediakeys.js), can you rename this to externs/msmediakeys.js?